internal/stage_7.go

```diff
@@ -20,8 +20,8 @@ func testGetFile(stageHarness *testerutils.StageHarness) error {
 	fileName := randomFileName()
 	fileContent := randomFileContent()
 
-	logger.Debugf("Creating file %s in %s", fileName, DATA_DIR)
-	logger.Debugf("File Content:\n%s", fileContent)
+	logger.Debugf("Creating file %q in %q", fileName, DATA_DIR)
```

Comment:
This line can use `%s`, the one below can use `%q` (since the exact value inside the file is important to know)